src/Symfony/Component/Messenger/Tests/Asynchronous/Routing/SenderLocatorTest.php

```diff
@@ -59,3 +92,20 @@ public function testItSupportsAWildcardInsteadOfTheMessageClass()
         $this->assertEquals(array($apiSender), $locator->getSendersForMessage(new SecondMessage()));
     }
 }
+
+class ChildDummyMessage extends DummyMessage
+{
+    private $extra;
```

Comment:
I don't see you using that extra so I guess it can be removed.

Reply:
Removed extra code.